File: app/src/main/java/android/emu6502/Emu6502Application.java
package android.emu6502;

import android.app.Application;

import com.facebook.stetho.Stetho;

public class Emu6502Application extends Application {

  @Override public void onCreate() {
    super.onCreate();
    Stetho.initialize(
        Stetho.newInitializerBuilder(this)
            .enableWebKitInspector(Stetho.defaultInspectorModulesProvider(this))
            .build());
  }
}

File: app/src/main/kotlin/android/emu6502/Emu6502Application.kt
package android.emu6502

import android.app.Application
import com.facebook.stetho.Stetho

class Emu6502Application : Application() {
  override fun onCreate() {
    super.onCreate()
    Stetho.initialize(Stetho.newInitializerBuilder(this)
        .enableWebKitInspector(Stetho.defaultInspectorModulesProvider(this))
        .build())
  }
}

